react/Geolocation.js

```diff
@@ -31,16 +35,33 @@ class Geolocation extends Component {
   }
 
   componentDidMount() {
+    this.startTime = Date.now()
     this.getPermissionStatus()
     if (this.props.askForGeolocation) {
       this.handleGetCurrentPosition()
     }
   }
 
-  componentDidUpdate(prevProps) {
+  unsubscribeAsyncTasks = () => {
+    this.unsubscribeGetCurrentPosition?.()
+    this.unsubscribeGetCurrentPosition = null
```

Comment:
assigning null to this might not be necessary but 🤷‍♂️

Reply:
what is the rationale for this? I can't see a scenario where it would need to be null again

Reply:
@wdsrocha just to remove the reference to the function, to maybe help the garbage collector clean it, in case the component for some reason has not been cleaned up yet

Reply:
I see, ty!